RELEASE_NOTES

```diff
@@ -1,3 +1,24 @@
+========================================================================
+Amazon CloudWatch Agent 1.247360.0 (2023-06-23)
+========================================================================
+
+Enhancements:
+* Support for voluntary collection of product experience metrics  (#766) (#771)
+
+========================================================================
+Amazon CloudWatch Agent 1.247359.0 (2023-04-20)
+========================================================================
+
+Enhancements and bug fixes:
+* Add log output when log files are deleted with auto_removal feature #636 (#659)
+* Add config mode for ModeWithIRSA to support using default credential provider with RUN_WITH_IRSA environment variable as True (#682)
+* Add -trimpath to go build command (#697)
+* Build Make Binary On Mac OS (#702)
+* CGO Enable 1 For Darwin arm64 (#714)
+* Only Enable CGO For Mac Build On Mac (#715)
+* Replace deprecated ioutil calls with os equivalent (#663)
```

Comment:
nit: I don't think these need to be in the release notes. They are build specific and do not matter to readers.

Reply:
cool, removing